src/main/scala/it/pagopa/interop/catalogprocess/api/impl/ProcessApiServiceImpl.scala

```diff
@@ -67,6 +67,19 @@ final case class ProcessApiServiceImpl(
     val result: Future[EService] = for {
       organizationId <- getOrganizationIdFutureUUID(contexts)
       clientSeed = Converter.convertToClientEServiceSeed(eServiceSeed, organizationId)
+      maybeEservice <- ReadModelQueries.listEServices(
+        eServiceSeed.name.some,
+        Seq.empty,
+        Seq(clientSeed.producerId.toString),
+        Seq.empty,
+        0,
+        1
+      )(readModel)
+      eServiceName = maybeEservice.results.map(_.name)
```

Comment:
Maybe it's more consistent this way?
`eServiceName = maybeEservice.results.headOption.map(_.name)`